src/GeoTables.jl

```diff
@@ -31,14 +31,14 @@ specify the layer of geometries to read within the file.
 - `*.geojson` via GeoJSON.jl
 - Other formats via ArchGDAL.jl
 """
-function load(fname, layer=0)
+function load(fname; layer=0, kwargs...)
```

Comment:
Awesome that you added kwargs, we should add this detail to the docstring of load and save.

Reply:
Yes, it is quite useful. I needed that flexibility to show that we can read from `.geojson`, process with `Meshes.jl`, and then write to `.shp`. I can add some details.

Reply:
It would be awesome to add some details to the docstring, thanks for these great improvements.

Reply:
Thank you @ErickChacon for adding examples to the README. We also need to update the docstrings of load and save to reflect the new possibility of forwarding kwargs.